include/deal.II/lac/trilinos_sparse_matrix.h

```diff
@@ -1475,15 +1475,14 @@ namespace TrilinosWrappers
      * Source and destination must not be the same vector.
      *
      * This function can be called with several different vector objects,
-     * namely TrilinosWrappers::VectorBase, TrilinosWrappers::MPI::Vector as well
-     * as deal.II's own vector classes Vector<double> and
-     * LinearAlgebra::distributed::Vector<double>.
+     * namely TrilinosWrappers::MPI::Vector as well as deal.II's own vector
+     * classes Vector<double> and LinearAlgebra::distributed::Vector<double>.
      *
-     * Note that both vectors have to be distributed vectors generated using
-     * the same Map as was used for the matrix in case you work on a
-     * distributed memory architecture, using the interface in the
-     * TrilinosWrappers::VectorBase class (or one of the two derived classes
-     * Vector and MPI::Vector).
+     * When using a vector of type TrilinosWrappers::MPI::Vector, both vectors
+     * have to be distributed vectors generated using the same Map as was used
```

Comment:
This issue (like the other nitpicks) was here before you edited things but I am not sure I understand this sentence and we should take this opportunity to fix things. Does `Map` refer to an `Epetra_Map`? Should we just refer to `IndexSet`s since the methods taking `Epetra_Map` or `Epetra_BlockMap` are deprecated?